app/models/spotlight/exhibit.rb

```diff
@@ -20,7 +20,7 @@ class Exhibit < ActiveRecord::Base
 
     extend FriendlyId
     friendly_id :title, use: [:slugged, :finders]
-    validates :title, presence: true
+    validates :title, presence: true, if: -> { I18n.locale == I18n.default_locale }
```

Comment:
I'm adding this because otherwise we were getting a validation error saving translations (through the exhibit via `nested_attributes`) and .

However; this can have the affect of allowing a site administrator to empty out the Exhibit's title if they are on that page in a non-default locale).  There are a few things we could do about this including changing the validation or disabling the normal exhibit label input when viewing in a non-default locale), but I'm not sure how much we would need to worry about it.

I would be happy to break dealing w/ this out into a separate ticket (although I think this change now would be useful because the form is effectively broken otherwise).

Reply:
@cbeer or @ggeisler have any thoughts on this?

Reply:
I would be fine with going with this approach now. But ticketing a more long-term solution to do validation for the translated values seems like a good idea.

Reply:
Just to make sure I'm being clear, I'm not suggesting we do any validations for the non-default language. I'll add more details in a new ticket shortly.

Reply:
Oh, I think I see. `presence` only needs to be enforced in the default locale because if it's not there in the translated language we just use the default locale version.